fastboot-tests/index-test.js

```diff
@@ -0,0 +1,49 @@
+var expect = require('chai').expect;
+
+function serializeMediaQueries($) {
+  let data = {};
+
+  $('#media-queries dt').toArray().forEach(function(dt) {
+    data[$(dt).text().trim()] = $(dt).next().text().trim();
+  });
+
+  return data;
+}
+
+describe('index', function() {
+
+  it('renders', function() {
+    return this.visit('/')
+      .then(function(res) {
+        var $ = res.jQuery;
+        // var response = res.response;
```

Comment:
left over from blueprint, to show how one could access the raw response. But not needed here, so will remove